intel_extension_for_transformers/backends/neural_engine/graph/scripts/convert_llama.py

```diff
@@ -951,8 +951,8 @@ def write_file_header(self, params: Params) -> None:
         self.fout.write(struct.pack("i", 0))
         self.fout.write(struct.pack("i", 0))  # word_embed_proj_dim (for opt)
         self.fout.write(struct.pack("i", 0))  # do_layer_norm_before (for opt)
-        fout.write(struct.pack("i", 1)) #llama's bos_token_id is 1
-        fout.write(struct.pack("i", int(hparams.get("eos_token_id", -1))))
+        self.fout.write(struct.pack("i", 1)) #llama's bos_token_id is 1
```

Comment:
change this annotation into `# TODO, bos_token_id = 0 in https://huggingface.co/decapoda-research/llama-7b-hf/blob/main/config.json but bos_token_id = 1 in llama.cpp`